GrocifyApp.UI/src/pages/ShoppingLists/ShoppingLists.jsx

```diff
@@ -0,0 +1,84 @@
+import { useState, useEffect } from 'react';
+import { ReactSVG } from 'react-svg';
+import { useNavigate } from 'react-router-dom';
+
+//Internal components
+import Searchbar from '../../components/Searchbar';
```

Comment:
![image](https://github.com/Catarina98/Grocify/assets/36730010/df008b29-5f0d-4bea-b952-20def781f9f6)

review comments and unnecessary imports